### DISCUSSION TITLE
请教下，hexo s 生成的网页不显示图片怎么解决

### DISCUSSION THREAD

**Discussion by DuLangDuBuTianXia** [hexojs/hexo#5314] [Question / Answer]
## Check List

Please check followings before submitting a new feature request.

- [x] I have already read [Docs page](https://hexo.io/docs/)
- [x] I have already searched existing issues

**Comment by uiolee** [comment]
please provide more detail and information

**Comment by DuLangDuBuTianXia** [comment]
typro软件显示出图片，hexo就显示不出图片，有没有办法解决

**Reply by awaae001** [reply]
在本地模式下，您需要使用.webp文件格式的图片

而不是jpg/png

**Comment by uiolee** [comment]
https://hexo.io/docs/asset-folders.html
